CHANGELOG.md

```diff
@@ -26,6 +26,7 @@ We refer to [GitHub issues](https://github.com/JabRef/jabref/issues) by using `#
 - We added integration of the Library of Congress catalog as a fetcher based on the [LCCN identifier](https://en.wikipedia.org/wiki/Library_of_Congress_Control_Number). [Feature request 636 in the forum](http://discourse.jabref.org/t/loc-marc-mods-connection/636)
 
 ### Fixed
+- We fixed the adding of a new entry from DOI which led to a connection error [#2879](https://github.com/JabRef/jabref/issues/2897)
```

Comment:
Suggestion: Improve wording to make clear that now HTTPS is used to protect the privacy of the user.